css-canvas-medical/styles.css

```diff
@@ -0,0 +1,265 @@
+  font-style: normal;
+  font-weight: 500;
+  border: none;
+}
+
+.row-two {
+  position: relative;
+  background-color: grey;
+  margin-bottom: 3.5em;
+}
+
+.row-two > img {
+  object-fit: cover;
+  opacity: 0.7;
+  display: block;
+  width: 100%;
+}
+
+.intro {
+  color: white;
+  position: absolute;
+  top: 3em;
+  left: 4em;
+  width: 480px;
+  height: 280px;
+}
+
+.intro > h2 {
+  font-size: 33px;
+  font-family: 'Tiempos Headline', serif;
+  font-style: normal;
+  font-weight: normal;
+}
+
+p {
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+.intro > p {
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 1.5em;
+}
+
+.get-started > input {
+  position: absolute;
+  background-color: #053c5e;
+  color: white;
+  border-radius: 0.25em;
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  border: none;
+  width: 170px;
+  height: 50px;
+  font-size: 17px;
+  bottom: 7.5em;
+  left: 3.5em;
+}
+
+.row-three {
+  background-color: #f8f9fa;
+  width: 1000px;
+  height: 400px;
+  border-bottom: 0.1em solid #d6d6d6;
+}
+
+.content-holder {
+  width: 850px;
+  display: flex;
+  margin: 20px auto;
+  flex-wrap: wrap;
+  color: #6f6866;
+}
+
+.sub-heading {
+  text-align: center;
+  margin: auto;
+  height: 180px;
+  width: 550px;
+}
+
+h3 {
+  font-family: 'Tiempos Headline', serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 25px;
+}
+
+.sub-heading > p,
+.learn > p {
+  font-size: 14px;
+}
+
+.location {
+  margin: auto;
+  width: 301px;
+}
+
+.location > p {
+  font-size: 16px;
+}
+
+.row-four {
+  display: flex;
+  margin-top: 10px;
+  justify-content: space-between;
+  color: #6f6866;
+}
+
+.more > input {
+  background-color: #053c5e;
+  color: white;
+  border-radius: 0.25em;
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  border: none;
+  width: 150px;
+  height: 40px;
+  font-size: 14px;
+  position: relative;
+  top: 1.5em;
+}
+
+.row-five {
+  width: 1000px;
+  margin: auto;
+  display: flex;
+  height: 300px;
+  background-color: #585856;
+  margin-top: 1.5em;
+}
+
+.address {
+  width: 400px;
+  position: relative;
+  top: 5.5em;
+  left: 3.5em;
+}
+
+.address > img {
+  width: 110px;
+  margin-bottom: 1.5em;
+}
+
+.address > p {
+  font-size: 12px;
+  line-height: 0.5em;
+  color: white;
+}
+
+.links {
+  width: 800px;
+  display: flex;
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+}
+
+span > a {
+  text-decoration: none;
+  font-family: 'Avenir', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  color: #6f6866;
+}
+
+.links a {
+  color: white;
+}
+
+.company {
+  top: -2em;
+  left: 6.5em;
+}
+
+.company,
+.product {
+  margin: auto;
+  position: relative;
+  color: white;
+}
+
+h5 {
+  font-size: 15px;
+  position: relative;
+  bottom: -0.5em;
+}
```

Comment:
Try to use the box model instead of positioning for adjustments like these.

Reply:
![Screen Shot 2021-01-31 at 3 21 00 PM](https://user-images.githubusercontent.com/74885918/106401266-000efc80-63d8-11eb-8651-2d72efdb3537.png)